crates/dbsp/src/algebra/present.rs

```diff
@@ -95,8 +95,7 @@ impl MulAssign<&'_ Present> for Present {
     fn mul_assign(&mut self, _rhs: &Self) {}
 }
 
-impl MulByRef<Present> for i32
-{
+impl MulByRef<Present> for i32 {
```

Comment:
we should probably add the checked weights too.
and we should probably run tests with checked weights